readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://<AgnieszkaWw>.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://<AgnieszkaWw>.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="930" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/b061a022-2631-4eee-a3a7-a4fb9fd198ee">